packages/kbn-sharedux-components/src/empty_state/no_data_views/no_data_views.test.tsx

```diff
@@ -11,19 +11,22 @@ import { ReactWrapper } from 'enzyme';
 
 import { mountWithIntl } from '@kbn/test-jest-helpers';
 import { EuiButton } from '@elastic/eui';
+import {
+  SharedUXServicesProvider,
+  SharedUXServices,
+  mockServiceFactories,
+} from '@kbn/sharedux-services';
 
-import { ServicesProvider, SharedUXServices } from '../../../services';
-import { servicesFactory } from '../../../services/mocks';
 import { NoDataViews } from './no_data_views';
 
 describe('<NoDataViewsPageTest />', () => {
   let services: SharedUXServices;
   let mount: (element: JSX.Element) => ReactWrapper;
 
   beforeEach(() => {
-    services = servicesFactory();
+    services = mockServiceFactories.servicesFactory();
```

Comment:
If the module is called `somethingServiceFactories`, should the method be called `services` rather than `servicesFactory`?

Reply:
The singleton is a collection of all of the factories, and the `servicesFactory` universal factory for all of them.  I think the naming could definitely use some work.